index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Author" content="Shubh M Das">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+
+<body>
+   <header>
+        <div class="container">
+            <div id="branding">
+                <h3>My Logo</h3>
+            </div>
+            <nav>
+                <ul>
+                 <li><a href="#">Portfolio</a></li> 
+                 <li><a href="#">About</a></li>
+                 <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
```

Comment:
- [ ] Please, consider that you are building the mobile version of your portfolio. That is why you have to implement the hamburger button. Please, consider placing it on the right side of your toolbar section.

> Here is your version
<img width="388" alt="Screen Shot 2022-02-10 at 15 08 02" src="https://user-images.githubusercontent.com/49064106/153395544-8d0de3c9-4bed-48dd-afb9-59fc33c0c1da.png">

> Here is the Fifma design
<img width="321" alt="Screen Shot 2022-02-10 at 15 14 24" src="https://user-images.githubusercontent.com/49064106/153395991-692f4158-aa9e-48d1-b427-cd1e754b4f74.png">